packages/link/src/Link.js

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import Tooltip from "@crave/farmblocks-tooltip";
+import { fontSizes } from "@crave/farmblocks-theme";
+
+import linkTypes from "./constants/linkTypes";
+import Container from "./styledComponents/Link";
+
+export default class Link extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showTooltip: false
+    };
+
+    this.mouseOver = this.mouseOver.bind(this);
+    this.mouseLeaves = this.mouseLeaves.bind(this);
+  }
+
+  render() {
+    const { children, ...props } = this.props;
+    return (
+      <Container {...props}>
+        {props.disabled && (
+          <div
```

Comment:
This div feels hackish.
Maybe remove this hit box and replace the ``<a>`` tag with a ``<span>`` tag when disabled, and attach the over/leave events to that tag.
By changing the markup we have a cleaner alternative to the current css-based way of preventing a click.

Reply:
Sure, this looks much better